tests/foreman/installer/test_installer.py

```diff
@@ -460,7 +460,6 @@ def test_foreman_rails_cache_store(sat_non_default_install):
     :BZ: 2063717, 2165092, 2244370
     """
     # Verify foreman-rails-cache-store option works
```

Comment:
@rmynar can you update the test steps? Specifically remove `Verify that foreman-redis package is not installed.`.

Reply:
fixed